studio/package.json

```diff
@@ -64,6 +64,7 @@
     "swr": "^1.2.1",
     "uniforms-bootstrap4": "^3.6.2",
     "uniforms-bridge-json-schema": "^3.6.2",
+    "uuid": "^8.3.2",
```

Comment:
do we need this? we have a helper for this `import { uuidv4 } from 'lib/helpers'`